usr/share/rear/verify/CDM/default/410_use_replica_cdm_cluster_cert.sh

```diff
@@ -25,12 +25,18 @@ while true; do
 done
 
 CDM_SUNOS_TAR=rubrik-agent-sunos5.10.sparc.tar.gz
+CDM_SOLARIS_TAR=rubrik-agent-solaris.sparc.tar.gz
+CDM_TAR_FILE=$CDM_SUNOS_TAR
 cd /tmp
-/usr/bin/curl $v -kLOJ https://${CDM_CLUSTER_IP}/connector/${CDM_SUNOS_TAR} 
-StopIfError "Could not download https://${CDM_CLUSTER_IP}/connector/${CDM_SUNOS_TAR}"
+/usr/bin/curl $v -fskLOJ https://${CDM_CLUSTER_IP}/connector/${CDM_TAR_FILE} 
+if [[ $? -gt 0 ]];  then
+    CDM_TAR_FILE=$CDM_SOLARIS_TAR
+    /usr/bin/curl $v -fkLOJ https://${CDM_CLUSTER_IP}/connector/${CDM_TAR_FILE} 
+fi
+StopIfError "Could not download cluster certificate extraction."
```

Comment:
I think in general you could make error messages more informative for the user
when you provide some variable values that tell the user what
in this case here that `cluster certificate extraction` thingy actually is e.g. like
```
StopIfError "Could not download cluster certificate extraction https://${CDM_CLUSTER_IP}/connector/${CDM_TAR_FILE}"
```

Reply:
Updated error message in 7b1e26b9b8c43934e46d8a1f95c8b85168a807df.